tests/src/tools/ReadyToRun.SuperIlc/PathHelpers.cs

```diff
@@ -31,6 +31,8 @@ static class PathExtensions
 
     internal static string OSExeSuffix(this string path) => (RuntimeInformation.IsOSPlatform(OSPlatform.Windows) ? path + ".exe" : path);
 
+    internal static char OSPathSeparator => (RuntimeInformation.IsOSPlatform(OSPlatform.Windows) ? ';' : ':');
```

Comment:
You don't need to define it yourself, there is Path.PathSeparator that contains this character.

Reply:
Nice simplification, thank you!